SAPHana/SAPHanaSR-ScaleOut.spec

```diff
@@ -91,6 +91,8 @@ install -m 0444 man/*.8.gz %{buildroot}/usr/share/man/man8
 install -m 0555 bin/* %{buildroot}/usr/sbin
 install -m 0555 test/SAPHanaSR-replay-archive %{buildroot}/usr/sbin
 install -m 0555 test/SAPHanaSR-filter %{buildroot}/usr/sbin
+install -m 0555 test/SAPHanaSR-SAPHanaSR-show-hadr-runtimes %{buildroot}/usr/sbin
+install -m 0555 test/SAPHanaSR-SAPHanaSR-show-hadr-log-coincidence %{buildroot}/usr/sbin
```

Comment:
Sorry, I overlooked it the first time - double 'SAPHanaSR-' in the name of the new scripts
SAPHanaSR-SAPHanaSR-show-hadr-runtimes - needs to be SAPHanaSR-show-hadr-runtimes
And the same for SAPHanaSR-SAPHanaSR-show-hadr-log-coincidence